lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -14,15 +14,15 @@
 // Header
 // ------
 .forum-nav-header {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
   display: table;
```

Comment:
Using `display: table` causes screenreaders to switch into table mode and read the data like a table. This should not be used for layout.